### PR TITLE
Update badges

### PR DESCRIPTION
**Pull request by Goooler**
[GitHub license](LICENSE.txt)
[Maven Central](https://central.sonatype.com/artifact/org.jetbrains.kotlinx/kotlinx-collections-immutable)

### CODE REVIEW

**Comment by Goooler** [comment]
https://github.com/Kotlin/kotlinx.collections.immutable/blob/f04cb64a7444765da090f29750773a4bbe611210/README.md?plain=1#L7

The current Maven Central link redirects to an invalid address, check https://central.sonatype.com/search?q=g:%22org.jetbrains.kotlinx%22%20%20a:%22kotlinx-collections-immutable%22&smo=true

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 [![Kotlin Alpha](https://kotl.in/badges/alpha.svg)](https://kotlinlang.org/docs/components-stability.html)
 [![JetBrains official project](https://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub) 
-[![GitHub license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0) 
+[![GitHub license](https://img.shields.io/github/license/kotlin/kotlinx.collections.immutable)](LICENSE.txt)
```
Comment (qurbonzoda) [inline README.md:5]: The library does not have `LICENSE.txt` file yet

**Review by qurbonzoda — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Goooler) [reply README.md:5]: https://github.com/Kotlin/kotlinx.collections.immutable/blob/master/LICENSE.txt

**Comment by qurbonzoda** [comment]
Thanks for the contribution!

**Pull request merged**